apps/render_cf.py

```diff
@@ -92,8 +92,13 @@ def get_job_parameters(item: str, items: str, job_spec: dict) -> dict:
     return job_parameters
 
 
-def get_step_for_batch_submit_job(task: dict, job_spec: dict) -> dict:
-    compute_environment = job_spec['compute_environment']['name']
+def get_step_for_batch_submit_job(task: dict) -> dict:
+    compute_environment = None
+    if 'compute_environment' in task:
+        if 'import' in task['compute_environment']:
+            compute_environment = task['compute_environment']['import']
+        else:
+            compute_environment = task['compute_environment']['name']
```

Comment:
We're requiring that each task always includes a `compute_environment`, right? If yes, then we could just simplify like this and let `KeyError` be raised if task doesn't include `compute_environment`:

```suggestion
    if 'import' in task['compute_environment']:
        compute_environment = task['compute_environment']['import']
    else:
        compute_environment = task['compute_environment']['name']
```